src/middlewares/is-admin.ts

```
@@ -0,0 +1,26 @@
+      HttpStatusCodes.UNAUTHORIZED
+    );
+  }
+  if (user.role !== "admin") {
+    return c.json(
+      {
+        message: "Unauthorized: Authentication required",
+        success: false,
+        error: "INSUFFICIENT_PERMISSIONS",
+      },
+      HttpStatusCodes.UNAUTHORIZED
+    );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)